src/qt/sendcoinsdialog.h

```diff
@@ -54,6 +54,9 @@ public Q_SLOTS:
     void setBalance(const CAmount& balance, const CAmount& unconfirmedBalance, const CAmount& immatureBalance,
                     const CAmount& watchOnlyBalance, const CAmount& watchUnconfBalance, const CAmount& watchImmatureBalance);
 
+Q_SIGNALS:
+    void coinsSent(QString);
```

Comment:
`const QString& txid` so that it's clear what the argument is?

Reply:
Done.